Add files via upload

mariannaperri opened the pull request:
Buongiorno, la ringrazio per la tempestività delle sue risposte alle mail precedenti. Ho caricato qua una bozza dei tecnici in vista dell'esame per capire se io stia andando nella giusta direzione. Ne approfitto per augurarle buone feste. Marianna Perri

strumet commented:
Ciao Lorenzo,
per aggiungere i tuoi file al repository del corso questi devono essere denominati come richiesto nelle indicazioni degli esercizi. Devi pertanto correggere la denominazione dei seguenti file:

- ex/vectors/886457 _01.svg (rimuovere lo spazio dopo il numero di matricola)
- ex/vectors/IMG_0676 3.JPG
- ex/vectors/IMG_2859 3.JPG

strumet commented:
file aggiunti :+1:

strumet commented:
Ciao Lorenzo,
devi semplicemente aggiornare il tuo repository locale.
Per farlo è sufficiente aprire il tuo repository locale tramite il terminale e digitare:
`git pull upstream master`

mariannaperri committed:
Add files via upload

Buongiorno, la ringrazio per la tempestività delle sue risposte alle mail precedenti. Ho caricato qua una bozza dei tecnici in vista dell'esame per capire se io stia andando nella giusta direzione. Ne approfitto per augurarle buone feste. Marianna Perri

strumet commented:
Ciao Marianna,
ti ringrazio per gli auguri che contraccambio sentitamente.
Fino alla prossima settimana non potrò effettuare la revisione per cui ti chiedo di avere un po' di pazienza. Appena avrò la possibilità guarderò il tuo lavoro e riceverai notifica via mail dell'avvenuta revisione.
A presto,
mf

strumet commented:
Ciao Marianna,

nell'immagine allegata troverai gli elementi da correggere più vistosi: si tratta di errori che è necessario correggere al fine di rispettare le indicazioni richieste dal tema d'esame che ti invito a rileggere con attenzione dalla sezione "Progetto d'esame" su BeeP.

Una volta sistemate queste parti e tracciate le sezioni (almeno 2, meglio 3 o 4) potrò entrare nel merito della correttezza del disegno: per adesso mi sembra che ci sia una certa disomogeneità grafica (definisci una regola chiara ed univoca per l'uso dei retini degli spessori e dei colori delle linee) ed alcune incongruenze (i pilastri sono distinti rispetto ai muri solo in uno dei piani).

Se dopo aver fatto le modifiche necessarie vuoi fare altre revisioni è sufficiente sostituire il pdf (rimuovi il file esistente e carica il nuovo mantenendo lo stesso nome) dal tuo repository in activities/5 ed effettuare un nuovo pull request.

Saluti,
mf

![913482](https://user-images.githubusercontent.com/6952124/71879869-5eaf6b80-312f-11ea-9efa-3b80e2716e4d.png)
